contracts/ERC721M.sol

```diff
@@ -365,6 +372,21 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         );
     }
 
+    /**
+     * @dev Returns mint currency address.
+     */
+    function getMintCurrency() external view returns (address) {
+        return _mintCurrency;
+    }
+
+    /**
+     * @dev Sets the mint currency used for payment.
+     */
+    function setMintCurrency(address mintCurrency) external onlyOwner {
+        _mintCurrency = mintCurrency;
+        emit SetMintCurrency(mintCurrency);
+    }
```

Comment:
thinking about setting mintCurrency again, is there a need for it to be per stage?

Reply:
Not for the first version, since we're just testing it out as a feature to see how many projects want it before adding more complexity to it